src/features/culturalSurvey/pages/CulturalSurveyIntro.tsx

```diff
@@ -25,6 +26,7 @@ const FAQTouchableLinkProps = {
 
 export const CulturalSurveyIntro = (): React.JSX.Element => {
   const { questionsToDisplay: initialQuestions } = useCulturalSurveyContext()
+  const { dispatch } = useNavigation()
```

Comment:
Tu peux directement utiliser `reset` sans `dispatch`. Aussi, tu peux mieux typer ce hook, en utilisant `UseNavigationType` de `'features/navigation/RootNavigator/types'` : 
```suggestion
  const { reset } = useNavigation<UseNavigationType>()
```